confluent_kafka/src/confluent_kafka.c

```diff
@@ -1324,24 +1324,37 @@ static void log_cb (const rd_kafka_t *rk, int level,
  * Clear Python object references in Handle
  */
 void Handle_clear (Handle *h) {
-	if (h->error_cb)
-		Py_DECREF(h->error_cb);
+    if (h->error_cb)
```

Comment:
Use 8 spaces for indent.
Put open-brace on same line as `if`:
`if (h->error_cb) {`

Reply:
Updated, please take another look.